summary41.txt

```diff
@@ -0,0 +1,99 @@
+Bugfixes Reset selected mission before follower training
+Balance Change long stick deconstruction recipe to not require cutting tool
+Bugfixes Crop locust spawn fix
+Interface Add hint for toggle base trait in wish_mutate menu
+Content Rag bundles
+Features Breaking long sticks as an item action
+Bugfixes Remove irremovable mods after the gun is destroyed by fire
+Balance Remove junk from plain popcorn
+Bugfixes Fix placement of basement stairs
+Infrastructure Handle UTF-8 string when format safemode message
+Infrastructure Handle using consoles with no map data
+Content Added an electric clay kiln, portable and vehicle
+Features Make chickens and other small domestic birds tameable (Chickenfeed)
```

Comment:
`tamable`

Reply:
Ups, stop the review please. I'll fix my spell checker and update PR.

Reply:
No worries 😄 these 3 were the only ones I found.

You might want to change uses of "json" to all caps as well while you're at it btw